### PR TITLE
fix equalizer effect spectrum view lag

### LINKED ISSUE #2736: Equalizer graph lag
Equalizer graph is lagging at ~2 fps. When I hold mouse on one of the points (1-8) then graph shows very smoothly. I believe it should always be smooth.

### PR DESCRIPTION
**Pull request by serdnab**
I reduced the refresh time of the spectrum view managed by a timer.
fixes #2736

### CODE REVIEW

File: plugins/Eq/EqSpectrumView.h [hunk 1]
```diff
@@ -144,7 +144,7 @@ class EqSpectrumView : public QWidget
 		setFixedSize( 400, 200 );
 		QTimer *timer = new QTimer(this);
 		connect(timer, SIGNAL(timeout()), this, SLOT(update()));
-		timer->start(100);
+		timer->start(10);
```
Comment (Umcaruje) [inline plugins/Eq/EqSpectrumView.h]: I'm not sure we need a 100fps display. Why not set the interval to 20, which gives us about 50fps, which is pretty smooth.

Comment (liushuyu) [inline plugins/Eq/EqSpectrumView.h]: Just FYI, modern displays (monitors) often have a refresh rate at 60 Hz (60
FPS).

Comment (Spekular) [inline plugins/Eq/EqSpectrumView.h]: Unfortunately 60 doesn't divide nicely by 1000.

Comment (liushuyu) [inline plugins/Eq/EqSpectrumView.h]: In this case…… How about use `16` ? This is the closest value.

**Comment by BaraMGB** [comment]
There is another bug: the analyzer graph freeze when there is no signal. It's about `m_energy` which keeps the last value. In the original spectrum analyzer plugin `m_energy` goes to `nan` if there is no signal anymore.

screenshot_20160617_162718

**Comment by serdnab** [comment]
It is also common a 75hz refresh rate of the monitor, this gives 13.3 ms.
I already tried with 20 ms on that refresh rate (75hz) and the difference is noticeable between holding or not a point.

**Comment by Spekular** [comment]
If we're going above 60hz then 144hz is probably the most common AFAIK.

File: plugins/Eq/EqSpectrumView.h (hunk 1 above)
Comment (tresf) [inline plugins/Eq/EqSpectrumView.h]: @serdnab can you please address the questions above?  I agree with @Umcaruje 20 is fine.  It doesn't matter if your monitor is 200 fps, your eyes can't see it and for the purposes of a visual bouncing EQ 50 fps should be perfectly suitable.

**Comment by serdnab** [comment]
@Umcaruje @tresf done, i changes 10ms to 20ms

**Comment by tresf** [comment]
> set the interval to 20, which gives us about 50fps, which is pretty smooth.

👍

**Pull request merged**